### ISSUE TITLE
Migrating to the Arrow mono-repo

### ISSUE THREAD

**Issue by pdeffebach** [ExpandingMan/Arrow.jl#49]
Discussion on discourse [here](https://discourse.julialang.org/t/arrow-feather-and-parquet/28739/28?u=pdeffebach) indicated that there is nothing stopping us in theory from having all code live in Apache Arrow's mono-repo but have the Julia package be a subset of that. 

Is there a plan to make that the case? It would be nice for Julia's data-frame saving package be backed by a funded organization that hopes to ensure cross-platform use.

**Comment by aschmu** [comment]
Yes I think this is very important as dataframes doesn't yet have an implementation as efficient in Julia as you'd find in R (via data.table or dplyr) or python via pandas.
It would probably good to develop an alternative dataframe library backed by arrow that would be columnar by design and hence much more suited for high performance analytics.
Given Julia's succint syntax, it could be a first class citizen for data munging in a way that python couldn't given its metaprogramming abilities.
